### ISSUE TITLE
Add a keyboard shortcut for `Insert link`

### ISSUE THREAD

**Issue by SimonBrandner** [vector-im/element-web#19501]
### Your use case

![Screenshot_20211025_125909](https://user-images.githubusercontent.com/25768714/138683854-63631a29-518a-4a25-a2dc-42c2f08ec9b6.png)

This should have a keyboard shortcut as it's faster than using the mouse

### Additional context

https://github.com/vector-im/element-web/issues/18887#issuecomment-950784736

**Comment by UwUnyaa** [comment]
@alexanderstephan how about macOS? What would be the equivalent shortcut there, especially to avoid conflicts?

**Comment by alexanderstephan** [comment]
@UwUnyaa Currently, the aim is to use <kbd>Ctrl/Cmd</kbd> + <kbd>Shift </kbd> + <kbd>L</kbd>, more info at the end of the linked PR above. This should not cause any problems from my testings. The review from a UX designer is still pending, so this is still very much up for discussion.

**Comment by UwUnyaa** [comment]
From my understanding, these changes should be made in matrix-org/matrix-react-sdk, right? I can only find relevant code there, and I see the potential of this being the first issue I work on.

**Comment by t3chguy** [comment]
https://github.com/matrix-org/matrix-react-sdk/pull/7104 already handles this
